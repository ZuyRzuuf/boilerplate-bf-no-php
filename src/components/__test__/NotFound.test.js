import * as React from 'react';

import NotFound from '../NotFound';

describe('<NotFound /> suite', function() {
    it('renders correctly isolated component', () => {
        const wrapper = shallow(
            <NotFound />
        );
        expect(wrapper).toMatchSnapshot();
    });

    it('renders correctly HTML tree', () => {
        const wrapper = render(
            <NotFound />
        );
        expect(wrapper).toMatchSnapshot();
    });

    it('renders correctly components tree', () => {
        const wrapper = mount(
            <NotFound />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
