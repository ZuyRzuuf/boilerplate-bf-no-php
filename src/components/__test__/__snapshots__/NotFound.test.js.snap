// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> suite renders correctly HTML tree 1`] = `
<div>
  NotFound
</div>
`;

exports[`<NotFound /> suite renders correctly components tree 1`] = `
<NotFound>
  <div>
    NotFound
  </div>
</NotFound>
`;

exports[`<NotFound /> suite renders correctly isolated component 1`] = `
<div>
  NotFound
</div>
`;
